src/createServer.js

```diff
@@ -1,3 +1,41 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase/convertToCase');
+const { wrongTargetCase } = require('./errorMessages');
+const { getTextAndParams } = require('./getTextAndParams');
+const { getErrors } = require('./getErrors');
+
+function createServer() {
+  const server = http.createServer((req, res) => {
+    const { text, targetCase } = getTextAndParams(req.url);
+    const responseErrors = getErrors(text, targetCase);
+
+    res.setHeader('Content-Type', 'application/json');
+
+    const response = {
+      originalText: text,
+      targetCase,
+    };
+
+    try {
+      Object.assign(response, convertToCase(text, targetCase));
+    } catch {
+      if (targetCase) {
+        responseErrors.errors.push({ message: wrongTargetCase });
+      }
+    }
+
+    if (!responseErrors.errors.length) {
```

Comment:
Would be better to another logic
First, you try to handle the error - then you make a succusses logic